hugo/content/changelog/_index.md

```diff
@@ -7,6 +7,12 @@ weight: 5
 
 ### New Features
 
+ * blackbox_exporter is now available for blackbox probing of endpoints over HTTP, HTTPS, DNS, TCP and ICMP. More info about this exporter is available via their github repo: https://github.com/prometheus/blackbox_exporter
```

Comment:
```suggestion
 * blackbox_exporter is now available for Blackbox probing of endpoints over HTTP, HTTPS, DNS, TCP and ICMP. More info about this exporter is available via their github repo: https://github.com/prometheus/blackbox_exporter
```

Reply:
I know this is lowercase on their own site but it's a name, right?

Reply:
If it's referring specifically to the name of the product and not using the `blackbox_exporter` name, then yes, I'd agree capitalizing Blackbox would be a good idea. However, in this case, it's referring to the generalized description of the word "blackbox" meaning it's probing something without knowing what its real intention is other than getting a tcp response.